### ISSUE TITLE
Missing dependency nyholm/psr7

### ISSUE THREAD

**Issue by bheisig** [Baldinof/roadrunner-bundle#16]
Hi,

first of all: Thank you very much for your roadrunner bundle! Unfortunately, I discovered a potential bug.

### How to reproduce

Require your library:

~~~
composer require \
        --update-no-dev \
        baldinof/roadrunner-bundle
~~~

Afterwards, install and configure roadrunner. Then start roadrunner.

### Actual behavior

With current version `1.3.0` roadrunner is not able to start. Excerpt from the logs:

~~~
Error: [http]: unable to connect to worker: [2020-09-04T09:48:46.995637+02:00] console.ERROR: Error thrown while running command "'baldinof:roadrunner:worker'". Message: "Class 'Nyholm\Psr7\Factory\Psr17Factory' not found" {"exception":"[object] (Error(code: 0): Class 'Nyholm\\Psr7\\Factory\\Psr17Factory' not found at /path/to/app/var/cache/prod/app/ContainerFQvxROM/getWorkerCommandService.php:32)","command":"'baldinof:roadrunner:worker'","message":"Class 'Nyholm\\Psr7\\Factory\\Psr17Factory' not found"} []
~~~

### Expected behavior

Since composer package `nyholm/psr7` is only suggested but not required, roadrunner should start without any errors.

The previous version I've used (`1.2.2`) seems unaffected.

### Workaround

Install the missing dependency right after installing your library:

~~~
composer require \
        --update-no-dev \
        nyholm/psr7
~~~

**Comment by Baldinof** [comment]
Hi!

Thank you for using it, I really appreciate the feedback :)

On version 1.3 it should fallback on laminas-diactoros, thanks to [php-http/discovery](https://github.com/php-http/discovery).

I did not managed to reproduced it on a fresh installation.

I see in the error that you are in `prod` environment, did you try to clear the cache?

Otherwise can you show me your `composer.json` and the result of `composer show`

**Comment by Baldinof** [comment]
I finnally managed to reproduce, it happens after running `composer require annotations`, because of [sensio/framework-extra-bundle](https://github.com/sensiolabs/SensioFrameworkExtraBundle/blob/master/src/Resources/config/psr7.xml), which define services with possibly not installed classes.

I will push a fix soon !

**Comment by Baldinof** [comment]
Hi,

I released 1.3.1, can you confirm the issue is fixed?

**Comment by bheisig** [comment]
Hi,

Indeed, version `1.3.1` works as expected :) That was amazingly fast! Thank you very much.
